client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ContactDetails.tsx

```diff
@@ -16,10 +16,14 @@ const ContactDetails = (props: Props) => {
     const { errors } = useFormContext();
     const { index , interactedContact } = props;
 
+    const isStatusCompleted = interactedContact.contactStatus === 5;
+
     const formErrors = errors.form ? (errors.form[index] ? errors.form[index] : {}) : {};
     const formHasErrors = Object.entries(formErrors)
         .some(([key, value]) => value !== undefined);
 
+    const showErrorIcon = formHasErrors && !isStatusCompleted;
+
```

Comment:
Change the method to validate in the schema itself, look inside the schema for id or phone validation and how it's done there, this approach is not scalable, contact me if you need help

Reply:
Meaning ignore the validation if the status is 5

Reply:
fixed [here](client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningClinical.ts)

Reply:
client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningClinical.ts

Reply:
fixed here : client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactSection/Schemas/contactQuestioningClinical.ts